.gitignore

```diff
@@ -4,4 +4,5 @@ log/
 dist/
 .ghci
 .env
+UserQueriesSpec.hs.old
```

Comment:
@kinorius123 Is necessary to add this file to the `.gitignore`? If this file is only generated in your computer, could you please only delete it?